README.md

```diff
@@ -23,21 +23,21 @@ It is recommended to install python dependencies in a virtual environment:
 If SELinux is in use on the ansible control host, enable access to the
 `selinux` python module from the virtualenv:
 
-`ln -s /usr/lib64/python2.7/site-packages/selinux venv/lib/python2.7/site-packages/selinux`
+    ln -s /usr/lib64/python2.7/site-packages/selinux venv/lib/python2.7/site-packages/selinux
```

Comment:
Isn't our normal approach to this to use `--system-site-packages` when invoking `virtualenv`?

Reply:
It is our normal approach but system-site-packages appears to interfere with lots of other packages installed at system level with packages we want to install in the virtual environment.  This approach is also much easier and targeted since we are only really doing this for selinux AFAIK!